src/bootstrap/src/core/build_steps/llvm.rs

```diff
@@ -1141,17 +1141,17 @@ fn supported_sanitizers(
     }
 }
 
-struct HashStamp {
-    path: PathBuf,
-    hash: Option<Vec<u8>>,
+pub(super) struct HashStamp {
```

Comment:
Could you move this e.g. to `src/utils/helpers.rs` instead of exporting it from the LLVM module?

Reply:
Sure, done!